src/Quasar/QuasarF/Interpreter/Aff.purs

```diff
@@ -0,0 +1,226 @@
+{-
+Copyright 2016 SlamData, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-}
+
+module Quasar.QuasarF.Interpreter.Aff (Config, eval) where
+
+import Prelude
+
+import Control.Bind ((=<<))
+import Control.Monad.Aff (Aff, attempt)
+import Control.Monad.Aff.Class (class MonadAff, liftAff)
+import Control.Monad.Eff.Console (CONSOLE)
+import Control.Monad.Eff.Exception (Error, error)
+import Control.Monad.Reader.Class (class MonadReader, ask)
+
+import Data.Argonaut (Json, (.?))
+import Data.Argonaut as Json
+import Data.Bifunctor (bimap, lmap)
+import Data.Either (Either(..), either)
+import Data.HTTP.Method (Method(..))
+import Data.List (List(..), (:))
+import Data.List as List
+import Data.Maybe (Maybe(..), maybe, fromMaybe)
+import Data.MediaType (MediaType(..))
+import Data.MediaType.Common (applicationJSON)
+import Data.NaturalTransformation (Natural)
+import Data.Path.Pathy (Path, Rel, Abs, RelDir, Sandboxed, rootDir, file, dir, relativeTo, printPath, peel, runDirName, runFileName, (</>))
+import Data.String as Str
+import Data.StrMap as SM
+import Data.Tuple (Tuple(..), fst, snd)
+
+import Global (encodeURIComponent)
+
+import Network.HTTP.Affjax as AX
+import Network.HTTP.Affjax.Request as AXR
+import Network.HTTP.RequestHeader as Req
+import Network.HTTP.StatusCode (StatusCode(..))
+
+import Quasar.Paths as Paths
+import Quasar.QuasarF (QuasarF(..), runLDJSON, AnyPath, Content, Pagination)
+
+type Config = { basePath ∷ AX.URL }
+
+eval
+  ∷ ∀ m eff
+  . ( MonadReader Config m
```

Comment:
I'll do something similar to handle authentication too, although `MonadState` I guess. I might need to put that in before I get anywhere with integrating this in SD actually.